### PR TITLE
CA-1229 + CA-1339: Passthroughs for billing account management

### PR DESCRIPTION
**Pull request by MatthewBemis**
Note that this merges into a feature branch

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/firecloud/webservice/BillingApiService.scala [hunk 1]
```diff
@@ -51,14 +51,21 @@ trait BillingApiService extends FireCloudDirectives {
                   passthrough(s"$v2BillingUrl/$projectId/members", GET)
                 }
               } ~
```
Comment (MatthewBemis) [inline src/main/scala/org/broadinstitute/dsde/firecloud/webservice/BillingApiService.scala:53]: it's probably easier to view this w/o whitespace changes

**Review by s-rubenstein — APPROVED** [review]
LGTM

File: src/main/resources/swagger/api-docs.yaml [hunk 2]
```diff
@@ -341,10 +343,84 @@ paths:
             'application/json':
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/billing/v2/{projectId}/billingAccount:
+    put:
+      tags:
+        - BillingV2
+      summary: update billing account on billing project
+      description: update billing account on billing project
```
Comment (gpolumbo-broad) [inline src/main/resources/swagger/api-docs.yaml]:
```suggestion
      description: Update the Billing Account on the Billing Project and on each Workspace in the Billing Project
```

File: src/main/resources/swagger/api-docs.yaml [hunk 3]
```diff
@@ -341,10 +343,84 @@ paths:
...
+      requestBody:
+        description: update billing account request
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/UpdateRawlsBillingAccountRequest'
+        required: true
+      responses:
+        200:
+          description: Billing account successfully updated
```
Comment (gpolumbo-broad) [inline src/main/resources/swagger/api-docs.yaml]:
```suggestion
          description: Billing Account on Billing Project is successfully updated and process is started to update the Billing Account on each Workspace in the Billing Project
```

File: src/main/resources/swagger/api-docs.yaml [hunk 4]
```diff
@@ -341,10 +343,84 @@ paths:
...
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
+      x-passthrough: true
+      x-passthrough-target: rawls
+    delete:
+      tags:
+        - BillingV2
+      summary: deletes the billing account on billing project
+      description: deletes the billing account on billing project
```
Comment (gpolumbo-broad) [inline src/main/resources/swagger/api-docs.yaml]:
```suggestion
      description: Clears the Billing Account on the Billing Project and on each Workspace in the Billing Project
```

**Review by gpolumbo-broad — APPROVED** [review]
I realize that it would be reasonable for you to have copied these descriptions for the Orch swagger directly from the Rawls swagger, which means if there are wording changes, they should be changed in both places.  If that's the case, feel free to ignore my suggestions for now.

**Comment by MatthewBemis** [comment]
@gpolumbo-broad I adopted the suggestions here. They were lifted from Rawls so I will go back and update those as well the next time I touch Rawls

**Pull request merged**